DOC-1726 - Integrate connection wizard with the RC Quickstart (#2208)
RedisLabs/redislabs-docs: content/rc/rc-quickstart.md

    "world"
    ```

    See the [commmand list](https://redis.io/commands/) for a list of all Redis commands.

### Redis client

> Different programming languages use different connection clients to interact with Redis databases. Each client has its own syntax and installation process, so read the documentation for the client you are using before running your code.

Reviewer: ```suggestion
Different programming languages use different connection clients to interact with Redis databases, and each client has its own syntax and installation process. For help with a specific client, see the client's documentation.
```

> The connection wizard contains code snippets to connect to your database for the following programming languages:

Reviewer: ```suggestion
The connection wizard provides code snippets to connect to your database with the following programming languages:
```

- node.js using [ioredis](https://github.com/luin/ioredis#quick-start)
- .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
- Python using [redis-py](https://github.com/redis/redis-py#redis-py)
- Java using [Jedis](https://github.com/redis/jedis#jedis)

See the [client list](https://redis.io/docs/clients/) to view all Redis clients by language.

#### Code example (Python)

Here's how to connect to your database using the [redis-py](https://github.com/redis/redis-py#redis-py) library for Python.

1.  Install the Redis client if it is not already installed.

    ```sh
    $ sudo pip install redis
    ```

2.  In the connection wizard, under **Redis Client**, select Python from the **Select your client** menu.

3. Select **Copy** to copy the connection code for your database.

> 4. Paste the code into your program, and replace `<username>` and `<password>` with your username and password.

Reviewer: ```suggestion
4. Add the copied code to your program and replace `<username>` and `<password>` with your username and password.
```

    ```python
    import redis

    r = redis.Redis(
      host='<host>',
      port=<port>,
      password='<password>')

    # Redis commands
    r.set('hello', 'world')
    print(r.get('hello'))
    ```

5. Run the program.

    ```sh
    $ python example_redis.py
    world
    ```

### RedisInsight

> RedisInsight is a free Desktop GUI for Redis that is available for MacOS, Windows, and Linux.

Reviewer: ```suggestion
RedisInsight is a free Redis GUI that is available for MacOS, Windows, and Linux.
```

1. In the connection wizard, under **Redis Client**, select your operating system from the **Download RedisInsight** menu.

1. Select **Download** to download RedisInsight.

1. [Install RedisInsight]({{< relref "/ri/installing/install-redis-desktop" >}}).

> 1. Once RedisInsight is installed and open, select **Add Redis Database**.

Reviewer: ```suggestion
1. After you install and open RedisInsight, select **Add Redis Database**.
```

1. In the connection wizard, under **RedisInsight Desktop**, select **Copy** to copy the connection information.

> 1. In RedisInsight, paste the connection information into the **Host** field. RedisInsight will automatically populate the rest of the information needed to connect to the database with the default user.

Reviewer: ```suggestion
1. In RedisInsight, enter the copied connection information into the **Host** field. RedisInsight automatically populates the rest of the fields needed to connect to the database as the default user.
```
